regression/contracts/function-calls-03-1/test-enf.desc

```diff
@@ -0,0 +1,18 @@
+KNOWNBUG
+main.c
+--enforce-all-contracts
+^EXIT=10$
+^SIGNAL=0$
+^VERIFICATION FAILED$
+--
+--
+This confirms the accuracy of the postconditions of both f1 and f2. 
+
+Test should fail:
+The postcondition of f2 is incorrect, considering the recursion particularity. 
+
+Recursion:
+The base case for the recursive call to f2 provides different behavior than the common case (given the pre-conditions).
+
+Known bug 2:
+This test requires the "--unwind 20 --unwinding-assertions" flag for the cbmc call in "chain.sh", which is currently not handled.
```

Comment:
Didn't you find a way to update `chan.sh` to handle parameters for CBMC call as well?